tensorrt_llm/_torch/models/modeling_mistral.py

```
@@ -332,8 +447,56 @@ def get_mm_special_token_ids(self) -> torch.Tensor:
         ])
 
 
+class MistralCommonInputProcessor(Mistral3InputProcessor):
+
+    def __init__(
+        self,
+        model_path: str,
+        config: PretrainedConfig,
+        tokenizer: AutoTokenizer,
+        trust_remote_code: bool = False,
+        **kwargs,
+    ):
+        tokenizer = self.load_tokenizer(model_path, config=config)
+        super().__init__(model_path=model_path,
+                         config=config,
+                         tokenizer=tokenizer,
+                         trust_remote_code=trust_remote_code,
+                         model_type="mistral_large_3",
+                         **kwargs)
+
+    @staticmethod
+    def load_tokenizer(model_path: str,
+                       config: PretrainedConfig,
+                       checkpoint_format: str = "mistral_large_3"):
+        if checkpoint_format == "mistral_large_3":
+            try:
+                return MistralTokenizer.from_pretrained(model_path)
+
+            except ValueError:
+                logger.info(
```

Dumb question: Is falling back the right thing to do? Will this not result in silent failure (or poor accuracy)?